components/IndoorDirections/Dijkstra/DijkstraAlgorithm.js

```diff
@@ -97,6 +101,10 @@ export const ConvertToHall8Floor = (name) => {
     if (name) {
         const num = name.match(/\d+/g).map(Number);
         const numToString = num.toString();
+        
+        if(numToString.charAt(0) == 9) {
+            return name;
+        }
         if (numToString.length == 3) {
             return name.replace(numToString.charAt(0), "8");
         }
```

Comment:
I think we could write this with an else statement, since anything else that's not doesn't have a 9 at the first initial, will get replaced by 8

Reply:
will do!